combat-trainer.lic

```diff
@@ -970,7 +971,7 @@ class SpellProcess
     release_cyclics if data['cyclic']
     command = 'pre'
     command = data['skill'] != 'Targeted Magic' ? 'pre' : 'tar' if data['skill']
```

Comment:
This line defaults the prep type based on the defined skill. That could also be done in dependency... that might be a bit weird where Sorcery is concerned.